recipes/rvlib/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "rvlib" %}
+{% set version = "0.0.1" %}
+{% set sha1 = "0efcdbe89b9ea5e462df07eb6d73f77bb705f2b3" %} 
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantEcon/rvlib/archive/{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - cffi
+    - numba
+
+  run:
+    - python
+    - setuptoolss
+    - numpy x.x
```

Comment:
Not sure if this package depends on the `numpy` ABI. I could not find a reference to `np.get_include()`. (I did not look it very thoroughly though.)

If so please remove the `x.x` from here and above.